Applications/Telemetry/readme.md

```diff
@@ -1,36 +1,40 @@
 # Telemetry
-The Telemetry application is a core application provided by FaradayRF. Its basic function is to receive telemetry packets from [Proxy](../../proxy) and save them into a SQLite database. Doing so allows us to document the data as well as search it easily in the future with SQL queries. Telemetry is built upon a Flask server providing a simple API for searching the telemetry database file. Telemetry_worker operates on its own thread to independently query Proxy for new data.
+The Telemetry application is a core application provided by FaradayRF. Its basic function is to receive telemetry packets from [Proxy](../../proxy) and save them into a SQLite database. Doing so allows us to document the data as well as search it easily in the future with SQL queries. Telemetry is built with a Flask server providing a simple API for searching the telemetry database file. The `telemetry_worker` function operates on its own thread to independently query Proxy for new data which continually updates the database in the background.
 
 Basing Telemetry on a [Flask](http://flask.pocoo.org/) server allows it to provide a RESTful interface to the network interface from which other applications such as a user interface or [APRS](../aprs) can query the saved data using simple HTTP GET commands.
 
-[DB Browser for SQLite](http://sqlitebrowser.org/) is a great tool which allows you to manipulate the database and perform simple plotting. It's simple to use the program to export a CSV for plotting in Excel as well.
-
-![Faraday Applications](images/FaradayProxyBlocks.jpg)
+This also allows us to use a standard web browser for basic debugging. We can easily see telemetry data with a simple URL query as we will show you shortly. [DB Browser for SQLite](http://sqlitebrowser.org/) is also a great tool which allows you to manipulate the database and perform simple plotting. It's simple to use the program to export a CSV for plotting in Excel as well.
 
 ## Configuration
 
 Telemetry uses [telemetry.ini](telemetry.ini) to properly connect to Proxy and save data to a SQLite database. This configuration file contains the following:
 
-* `[flask]` Flask telemetry server section
- * `host` Flask hostname or IP address
- * `port` Network port to serve data
+* `[FLASK]` Flask telemetry server section
+ * `HOST` Flask hostname or IP address
+ * `PORT` Network port to serve data
 
-* `[database]` Telemetry SQlite database section
- * `filename` Database filename "x.db"
- * `schemaname` SQLite schema file to setup tables "x.sql"
+* `[DATABASE]` Telemetry SQlite database section
+ * `FILENAME` Database filename "x.db"
+ * `SCHEMANAME` SQLite schema file to setup tables "x.sql"
 
-* `[telemetry]` Telemetry application section
- * `units` Quantity of Faraday radios connected to computer
- * `unit0call` Callsign of first radio
- * `unit0id` Node ID of first radio
+* `[TELEMETRY]` Telemetry application section
+ * `UNITS` Quantity of Faraday radios connected to computer
```

Comment:
This is no different than the current implementation of APRS or Proxy. I've created [Issue #86](https://github.com/FaradayRF/Faraday-Software/issues/86) to document and address this in the future.